### ISSUE TITLE
Output consistency

### ISSUE THREAD

**Issue by nitsnatsnok** [lingdb/CoBL#182]
We are dealing with an overwhelming amount of special signs that is highly probable to cause display problems, as the brower has to use the system's installed fonts – and they do mostly not cover our needs. To try this, I changed my browser's standard font (and nothing else, especially not the font size!):
### Arial
![arial](https://cloud.githubusercontent.com/assets/2664162/15274122/0574e86c-1aac-11e6-8d9a-005b5dc9ac9c.png)
### Courier New
![courier new](https://cloud.githubusercontent.com/assets/2664162/15274137/279ce926-1aac-11e6-98c4-3334fe9e463d.png)
### DejaVu Sans
![dejavu sans](https://cloud.githubusercontent.com/assets/2664162/15274135/279c732e-1aac-11e6-8e29-c50c535cabbd.png)
### Liberation Sans
![liberation sans](https://cloud.githubusercontent.com/assets/2664162/15274136/279ca510-1aac-11e6-9209-32f4fa2d24b6.png)
### Monospace
![monospace](https://cloud.githubusercontent.com/assets/2664162/15274134/279c2964-1aac-11e6-900f-76c71afcea65.png)
### Times New Roman
![times new roman](https://cloud.githubusercontent.com/assets/2664162/15274133/279bf372-1aac-11e6-9e54-7797d9849a08.png)
### Ubuntu
![ubuntu](https://cloud.githubusercontent.com/assets/2664162/15274138/27a0b7fe-1aac-11e6-8b64-36ed2c5e7a74.png)

As you can see, the output varies for every single of these fonts, especially regarding special signs, and they are likeli to be displayed wrong. But it's not even just about special character representation: The leftmost lines appear from very thick to nearly invisible and some fonts produce a linebreak in the “Cog. Alias” field (see “Ad” class). Note that I tried this with Chromium 49.0.2623.108, which is a powerful up-to-date browser. I don't want to think about results from some outdated Internet Explorer.
IMHO we should try to get a consistent output for special signs, **not** depending on the user's installed fonts.

**Comment by runjak** [comment]
Yes, I agree - the two options I see for dealing with this would be:

* Making sure we have a selection of fonts set in the css so that browsers can smartly select appropriate fonts.
* Supplying the font to be used as a webfont to make sure it is available to the client.

There are some existing resources for webfonts, like a [collection](https://www.google.com/fonts) by google.
I think the big problem here would be figuring out which font to use based on which symbols are covered.

**Comment by runjak** [comment]
From investigating the css provided by bootstrap I infer that the currently used fonts should be these:

```
  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
  font-family: Menlo, Monaco, Consolas, "Courier New", monospace;
```

I'm a bit uncertain if and how webfont choices are overwritten by the browser settings,
but I wouldn't be surprised if client side settings can overwrite our font choices in any situation. This would mean that the font problem could not be completely solved as it would always be possible for some user to choose an unhelpful font.
